lib/invisible_captcha.rb

```diff
@@ -36,6 +42,16 @@ def init!
       # If enabled, you should call anywhere in of your layout the following helper, to inject the honeypot styles:
       #  <%= invisible_captcha_styles %>
       self.injectable_styles = false
+      
+      # IP check enabled by default
+      self.ip_enabled = true
+      
+      # A secret key setup in config settings. 'rake secret' will give you a good one.
+      # Rails.application not currently loading. Need to change the way this loads.
+      self.secret = 'Secret' # InvisibleCaptcha::SecretKeyFinder.new(Rails.application).find
+   
+      # Actions that exclude running the detect_spam method, usd mainly for view actons
+      self.exclude_detect_actions = ['new', 'edit']
```

Comment:
as commented above, this is a bit confusing API from my point of view... what about invisible_captcha occurrences for non-rest resources?